### PR TITLE
Task-56576: native jitsi integration on ios

### PR DESCRIPTION
**Pull request by wbenabdessalem**
We need to integrate native jitsi on ios version so that when we use jitsi, we will have native mobile and not responsive.

### CODE REVIEW

File: eXo/Sources/Controllers/HomePage/HomePageViewController.swift [hunk 1]
```diff
@@ -275,6 +287,14 @@ class HomePageViewController: eXoWebBaseController, WKNavigationDelegate, WKUIDe
             print("=============== Navigation Url : \(urlToSee)")
         }
         
+        // Open the Jitsi call.
+        
+        if request.url?.absoluteString.range(of:"?jwt=") != nil {
```
Comment (ahamdi) [inline eXo/Sources/Controllers/HomePage/HomePageViewController.swift]: Could we check on the Jitsi URL instead of checking just for the jwt token ? we may have integration with JWT for other applications

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: eXo/Sources/Controllers/HomePage/HomePageViewController.swift (hunk 1 above)
Comment (wbenabdessalem) [reply eXo/Sources/Controllers/HomePage/HomePageViewController.swift]: Done

File: eXo/eXoRelease.entitlements [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
```
Comment (ahamdi) [inline eXo/eXoRelease.entitlements:4]: Why did you add this file ?

File: eXo/Sources/defines.swift [hunk 3]
```diff
@@ -41,6 +41,12 @@ struct Config {
     
     static let kTableCellHeight: CGFloat = 80.0
     static let kTableHeaderHeight: CGFloat = 50.0
+    
+    // eXo Jitsi Server Path
+    static let JitsiServer = "https://meet.jit.si"
```
Comment (ahamdi) [inline eXo/Sources/defines.swift]: Do we need this variable?

File: eXo/Sources/Controllers/HomePage/HomePageViewController.swift [hunk 4]
```diff
@@ -603,3 +622,134 @@ extension HomePageViewController:WKDownloadDelegate {
...
+                jitsiMeetView.alpha = 0
+                self.pipViewCoordinator?.show()
+            }
+        }
+    }
+    
+    func openJitsiMeet(webview:WKWebView,url:URL){
+            cleanUp()
+            webViewDidClose(webview)
+            let username = PushTokenSynchronizer.shared.username!
```
Comment (ahamdi) [inline eXo/Sources/Controllers/HomePage/HomePageViewController.swift]: Couldn't we get the username from the main app instead of the PushTokenSynchronier ?

**Review by ahamdi — APPROVED** [review]
